### PR TITLE
#154551427 Add Notifications 

### PR DESCRIPTION
**Pull request by NaiRobley**
#### What does this PR do?
Add Notifications

#### Description of Task to be completed?
Whenever an event occurs that the user should be notified of, a snack-bar/toast including the message should be displayed to the user as shown in the mockups 

#### Where should the reviewer start?
[Custom Notification Component](https://github.com/AndelaOSP/wire/blob/ft-add-notifications-154551427/src/Components/CustomNotification/CustomNotification.Component.jsx)

#### What are the relevant pivotal tracker stories?
[#154551427](https://www.pivotaltracker.com/n/projects/2117172/stories/154551427/)

#### Screenshots (if appropriate)

Success notification
image

Error notification
image

### CODE REVIEW

File: __tests__/CustomNotification.test.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (danielwangai) [inline __tests__/CustomNotification.test.jsx]: Remove this empty line

Comment (NaiRobley) [reply __tests__/CustomNotification.test.jsx]: Removed

File: src/Components/CustomNotification/CustomNotification.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    background-color: map_get($white, 100) !important;
+
+    &>div {
+      &>span {
+        padding: 0 !important;
+        font-size: 1rem !important;
+      }  
+    }
+  }  
+
```
Comment (Mbarak-Mbigo) [inline src/Components/CustomNotification/CustomNotification.scss]: Remove extra line

File: src/Components/CustomNotification/CustomNotification.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
+@import '../../theme.scss';
+
+.snackbar {
+  margin-bottom: 2rem;
+  box-shadow: 0 8px 16px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
```
Comment (Mbarak-Mbigo) [inline src/Components/CustomNotification/CustomNotification.scss]: Use `rem` instead of `px`

**Comment by NaiRobley** [comment]
@Mbarak-Mbigo Changes made.

**Pull request merged**